neural_clbf/training/train_inverted_pendulum.py

```diff
@@ -109,6 +109,7 @@ def main(args):
         num_init_epochs=5,
         epochs_per_episode=100,
         barrier=False,
+        disable_gurobi=True,
```

Comment:
Let's not change the examples on the main branch.

Reply:
The problem with no changing the example is that new users like me will immediately hit an error and will need to find this pull request or [this issue](https://github.com/MIT-REALM/neural_clbf/issues/8#issuecomment-1362096595) to know how to fix it. I think it would be better to have an example that works right out of the box. In fact there is another change to `train_inverted_pendulum.py` I would like to push to avoid another error, but it has to do with matplotlib and not gurobi so I haven't added it to this PR

Reply:
That's a good point, but there's another consideration that the default behavior should reproduce the experiments in our publications that reference this code. Is there a way to add a more useful error message, or to update the README (maybe in the installation instructions) to make this clear to new users?

Reply:
This largely comes down to your preferences on how you want users to interact with your repo. Personally I would separate my first-user-experience examples from my paper-reproduction experiments such that a new user could come to this repo and immediately run examples without any knowledge of your paper. As for researchers trying to reproduce your results, I would have clear-yet-separate instructions in the README as to how to reproduce the paper experiments (e.g. specific commit hash/tag to checkout, specific command to run, etc.). If it's any help, I can share an example of how I've done this (it's actually a repo I am developing that compares/contrasts with neural_clbf; I haven't pushed it public yet but it's contained on MIT's github that I assume you have access to)

Reply:
Those are good points. Thanks for making them! I'll approve this PR and then make the docs changes (I might also experiment with auto-magically selecting CVXPY or gurobi based on available licenses)